src/tests/test_magic_cmd.py

```diff
@@ -9,10 +9,18 @@
 from sql.store import store
 from sql.inspect import _is_numeric
 from sql.display import Table, Message
+from sql.connection.connection import ConnectionManager
 
+jupysql_plugin_available = False
+try:
+    from jupysql_plugin.widgets import ConnectorWidget
+
+    jupysql_plugin_available = True
+except ImportError:
+    pass
 
```

Comment:
jupysql-plugin must be available in the CI (see [this](https://github.com/ploomber/jupysql/blob/424ff48e62d2e016c3f1ee3fe3f1e69ae83b0615/setup.py#L53)) since it's declared as a development dependency. so there's no need to do this `try .. except`

Reply:
[fixed](https://github.com/ploomber/jupysql/pull/783/files#diff-9188e75e824219dd191cdc62d6d19f2b15c4bd0cf564abe3646ea907c32cd61dR13) and add `jupyter_server` requirement [here](https://github.com/ploomber/jupysql/pull/783/files#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7R56-R57).

`jupysql_plugin` was already there but it uses [jupyter_server](https://github.com/ploomber/jupysql-plugin/blob/d26ff4a550c9897af73a4a617e909eafe451091e/jupysql_plugin/server_handlers/dashboard.py#L3) so I added it.